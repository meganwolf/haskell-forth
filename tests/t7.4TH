HELLO WORLD PEACE CONCAT2 .
HELLO WORLD PEACE CONCAT3 CR .