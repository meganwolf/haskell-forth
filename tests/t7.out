WORLDPEACE
HELLOWORLDPEACE
"Stack not empty"
[Id "HELLO"]